tests/base.py

```diff
@@ -8,6 +8,15 @@
 from datetime import timedelta
 
 from tap_tester.base_suite_tests.base_case import BaseCase
+from tap_tester.jira_client import JiraClient as jira_client
```

Comment:
Why is this being renamed?

Reply:
It seemed like camel case was primarily being used for class names and variables, functions, etc used snake case.  Is there a standard convention?

Reply:
> Is there a standard convention?

- https://peps.python.org/pep-0008/#class-names
  > CapWords
- https://peps.python.org/pep-0008/#function-and-variable-names
  > Function names should be lowercase, with words separated by underscores as 
    necessary to improve readability.
    Variable names follow the same convention as function names.

Reply:
So I think the confusion here is that you're calling the constructor of `JiraClient`, which to your point, is a function and maybe should be named like one.

But I think it's understood that constructors are special functions and we shouldn't rename the class import just to fit some naming convention

Reply:
Updated.

Reply:
For real this time.